etc/atalkd/main.c

```diff
@@ -547,7 +547,14 @@ static void as_timer(int sig _U_)
 		    }
 
 		    /* split horizon */
-		    if (rtmp->rt_iface == iface) {
+
+		    /* Makes the AsanteTalk bridge consistently start up in
+		     * AppleTalk Phase 2, and stop the Dayna bridge from
+		     * crashing GS/OS.
+		     */
+		    if (quirks && (rtmp->rt_iface != iface)) {
+		        continue;
+		    } else if (rtmp->rt_iface == iface) {
```

Comment:
This line is likely still executing when "quirks mode" is enabled.

Alter line to `} else if (!quirks && rtmp->rt_iface == iface) {`

Reply:
Ah, of course. The dangers of coding late at night.

Fixed.